app/views/case_contacts/show.html.erb

```diff
@@ -7,7 +7,7 @@
 
 <p>
   <strong>Casa case:</strong>
-  <%= @case_contact.casa_case_id %>
+  <%= @case_contact_number %>
```

Comment:
This can be rewritten as:
```
<%= @case_contact.casa_case.case_number %>
```
If `@case_contact` is defined in the controller action.

Reply:
This is the shape of @case_contact_number:

{"id":19,
"creator_id":5,
"casa_case_id":4,
"contact_type":"youth",
"other_type_text":null,
"duration_minutes":15,
"occurred_at":"2020-04-05T00:00:00.000Z",
"created_at":"2020-04-05T23:36:26.440Z",
"updated_at":"2020-04-05T23:36:26.440Z"
}

we do not have the casa_case object. Which is why we have stuff in the controller to get the information.